src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -190,7 +186,20 @@ export class AdditionalServicesUtils {
     const servicesToAdd = this.installedServicePkgs.filter((s) => s.added === false);
 
     this.installedServicePkgs = addedServices.sort((a, b) => a.type.localeCompare(b.type)).concat(servicesToAdd.sort((a, b) => a.type.localeCompare(b.type)));
-    this.sharedService.installedServicePkgs.next({ installed: this.installedServicePkgs, availableToInstall: this.availableServicePkgs });
+    if (from !== 'additional-services') {
+      const service = { name: '', added: false, isEnabled: false, isInstalled: false, process: from };
+      const serviceDetail = this.installedServicePkgs.find(s => ['bucket', 'dispatcher', 'notification', 'management'].includes(s.process));
+      if (serviceDetail) {
+        service.isEnabled = ["shutdown", "disabled", "installed"].includes(serviceDetail.state) ? false : true;
```

Comment:
this can be optimized like
`!["shutdown", "disabled", "installed"].includes(serviceDetail.state);`